src/content/apis/sports/api-football.md

```diff
@@ -0,0 +1,10 @@
+---
+name: API Football
+description:  API to access all API endpoints, which can get information about Football Leagues & Cups
+categories:
+  - sports
+features:
+  https: true
+url: https://www.api-football.com/documentation-v3
+logo: https://www.api-football.com/public/img/home1/hero-banner.png

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.